src/core/Ros.js

```diff
@@ -4,6 +4,7 @@
  */
 
 import socketAdapter from './SocketAdapter.js';
+import { isRosbridgeActionFeedbackMessage, isRosbridgeActionResultMessage, isRosbridgeCallServiceMessage, isRosbridgeCancelActionGoalMessage, isRosbridgePublishMessage, isRosbridgeSendActionGoalMessage, isRosbridgeServiceResponseMessage, isRosbridgeStatusMessage } from '../types/protocol.js';
```

Comment:
Don't we have some limit on line length? 294 characters seems a bit excessive...

Reply:
#1003